src/index.html

```diff
@@ -13,5 +13,142 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <h2>Personal information</h2>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="wrapper"> <legend>Personal information</legend>
+        <label class="personal">
+          Surname: 
+          <input 
+            type="text" 
+            name="surname"  
+            autocomplete="off"
+          >  
+        </label>
+        <label class="personal">
+          Name:
+          <input 
+            type="text" 
+            name="name" 
+            autocomplete="off"
+          >  
+        </label>
+        <label class="personal">
+          How old are you?
+          <input
+            type="number" 
+            name="old" 
+            max="100" 
+            min="1" 
+            value="12"
+          >  
+        </label>
+        <label class="personal">
+          Full date of birth:
+          <input
+            type="date" 
+            name="birth"
+            >
```

Comment:
```suggestion
          <input
            type="date" 
            name="birth"
          >
```
Closing tag should be under the opening